modules/extension/xsd/xsd-wps/src/main/java/org/geotools/wps/v2_0/bindings/ComplexDataTypeBinding.java

```diff
@@ -0,0 +1,104 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2021, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.wps.v2_0.bindings;
+
+import javax.xml.namespace.QName;
+import net.opengis.wps20.ComplexDataType;
+import net.opengis.wps20.Wps20Factory;
+import org.geotools.wps.v2_0.WPS;
+import org.geotools.xsd.AbstractComplexEMFBinding;
+import org.geotools.xsd.ElementInstance;
+import org.geotools.xsd.Node;
+
+/**
+ * Binding object for the type http://www.opengis.net/wps/2.0:ComplexDataType.
+ *
+ * <p>
+ *
+ * <pre>
+ *  <code>
+ *  &lt;?xml version="1.0" encoding="UTF-8"?&gt;&lt;complexType name="ComplexDataType" xmlns="http://www.w3.org/2001/XMLSchema"&gt;
+ *
+ *  		&lt;complexContent&gt;
+ *
+ *  			&lt;extension base="wps:DataDescriptionType"&gt;
+ *
+ *  				&lt;sequence&gt;
+ *
+ *  					&lt;annotation&gt;
+ *
+ *  						&lt;documentation&gt;
+ *
+ *  							Placeholder for schema extensions to WPS complex data.
+ *
+ *  						&lt;/documentation&gt;
+ *
+ *  					&lt;/annotation&gt;
+ *
+ *  					&lt;any maxOccurs="unbounded" minOccurs="0" namespace="##other" processContents="lax"/&gt;
+ *
+ *  				&lt;/sequence&gt;
+ *
+ *  			&lt;/extension&gt;
+ *
+ *  		&lt;/complexContent&gt;
+ *
+ *  	&lt;/complexType&gt;
+ *
+ *   </code>
+ *  </pre>
+ *
+ * @generated
+ */
+public class ComplexDataTypeBinding extends AbstractComplexEMFBinding {
+
+    public ComplexDataTypeBinding(Wps20Factory factory) {
+        super(factory);
+    }
+
+    /** @generated */
+    @Override
+    public QName getTarget() {
+        return WPS.ComplexDataType;
+    }
+
+    /**
+     *
+     * <!-- begin-user-doc -->
+     * <!-- end-user-doc -->
+     *
+     * @generated modifiable
+     */
+    @Override
+    public Class getType() {
+        return ComplexDataType.class;
+    }
+
+    /**
+     *
+     * <!-- begin-user-doc -->
+     * <!-- end-user-doc -->
+     *
+     * @generated modifiable
+     */
+    @Override
+    public Object parse(ElementInstance instance, Node node, Object value) throws Exception {
+
+        // TODO: implement and remove call to super
+        return super.parse(instance, node, value);
```

Comment:
Either remove this method or add the missing implementation?

Generally speaking, I'm wondering why the implementation is completely separate from the WPS 1.0 one. 
In other services, when the two protocols are very similar, the bindings do share code, and sometimes they are a straight extension of each other, by subclassing (e.g., see SLD 1.1 vs SLD 1.0, WFS 1.1 vs WFS 1.0)

Reply:
Class removed. Use org.geotools.xsd.ComplexEMFBinding in WPSConfiguration to reduce number of bindings class